tutorials/inputs/controllers_gamepads_joysticks.rst

```diff
@@ -57,8 +57,10 @@ There are 3 ways to get input in an analog-aware way:
     # The line below is similar to `get_vector()`, except that it handles
     # the deadzone in a less optimal way. The resulting deadzone will have
     # a square-ish shape when it should ideally have a circular shape.
-    var velocity = Vector2(Input.get_action_strength("move_right") - Input.get_action_strength("move_left"),
-		Input.get_action_strength("move_back") - Input.get_action_strength("move_forward")).clamped(1)
+    var velocity = Vector2(
+            Input.get_action_strength("move_right") - Input.get_action_strength("move_left"),
+            Input.get_action_strength("move_back") - Input.get_action_strength("move_forward")
+    ).clamped(1.0)
```

Comment:
```suggestion
    ).limit_length(1.0)
```

Clamped was deprecated in 3.x and the class reference recommended using limit_length instead